### PR TITLE
Register Subdomain: raghavyuva

### PR DESCRIPTION
**Pull request by raghavyuva**
Added the domain: raghavyuva

### CODE REVIEW

File: domains/raghavyuva.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "raghavyuva",
+        "email": "vikramnbhat15@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "185.199.108.153"
```
Comment (WilliamDavidHarrison) [inline domains/raghavyuva.json:9]: I think you meant to use an `A` record for this?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/raghavyuva.json (hunk 1 above)
Comment (raghavyuva) [reply domains/raghavyuva.json]: CNAME field has been updated,kindly check it out

File: domains/raghavyuva.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "185.199.108.153"
+        "CNAME": "https://raghavyuva.github.io/portfolio/"
```
Comment (WilliamDavidHarrison) [inline domains/raghavyuva.json:9]: Sorry, `CNAME` records cannot contain `https://` or `/`. Please change this to just `raghavyuva.github.io`

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/raghavyuva.json (hunk 2 above)
Comment (raghavyuva) [reply domains/raghavyuva.json]: changed

File: domains/raghavyuva.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "https://raghavyuva.github.io/portfolio/"
+        "CNAME": "raghavyuva.github.io/portfolio/"
```
Comment (WilliamDavidHarrison) [inline domains/raghavyuva.json:9]: Please remove the `/portfolio/`

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/raghavyuva.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "raghavyuva.github.io/portfolio/"
+        "CNAME": "raghavyuva.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/raghavyuva.json:9]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/raghavyuva.json (hunk 4 above)
Comment (raghavyuva) [reply domains/raghavyuva.json:9]: That's why It is github.com/raghavyuva/portfolio and not github.com/raghavyuva

Comment (WilliamDavidHarrison) [reply domains/raghavyuva.json:9]: Please rename your portfolio repo to raghavyuva.github.io

Comment (raghavyuva) [reply domains/raghavyuva.json:9]: Changed,

Comment (raghavyuva) [reply domains/raghavyuva.json]: kindly check this out.

**Review by wdhdev — APPROVED** [review]

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**